README.md

```diff
@@ -16,24 +16,41 @@
 
 KubeVela is the platform engine to create *PaaS-like* experience on Kubernetes, in a scalable approach.
 
+For platform builders, KubeVela serves as a framework that empowers them to create user friendly yet highly extensible platforms at ease.
```

Comment:
Copy paste from introduction doc is not very help (and created duplicated content).

A good example for readme: https://github.com/Azure/Orkestra

Reply:
reverted